src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Component {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Component {
```

Comment:
According to the checklist, custom annotations should have the `@Target` annotation to specify where this annotation can be used. For example, if `@Component` is intended to be used on classes, it should have `@Target(ElementType.TYPE)`. Please add the `@Target` annotation to the `@Component` annotation.

Reply:
ok, done